x-pack/plugins/dashboard_enhanced/public/services/drilldowns/dashboard_drilldowns_services.ts

```diff
@@ -27,13 +26,16 @@ declare module '../../../../../../src/plugins/ui_actions/public' {
 }
 
 export class DashboardDrilldownsService {
-  async bootstrap(
-    core: CoreSetup<{ drilldowns: DrilldownsStartContract }>,
-    plugins: SetupDependencies
-  ) {
+  async bootstrap(core: CoreSetup<StartDependencies>, plugins: SetupDependencies) {
```

Comment:
We need to remove this `async`.

```suggestion
  bootstrap(core: CoreSetup<StartDependencies>, plugins: SetupDependencies) {
```